lib/kitchen/verifier/pester.rb

```diff
@@ -97,16 +147,17 @@ def init_command
       # required, then `nil` will be returned.
       #
       # @return [String] a command string
-      def prepare_command; end
+      def prepare_command
+        info("Preparing the SUT and Pester dependencies...")
+        really_wrap_shell_code(install_command_script)
+      end
 
       # Generates a command string which will invoke the main verifier
       # command on the prepared instance. If no work is required, then `nil`
       # will be returned.
       #
       # @return [String] a command string
       def run_command
-        return if local_suite_files.empty?
```

Comment:
Do we want to run even if there are no test suite files to run?

Reply:
Yes IMO. I want to make sure Pester runs, even if there's no test to execute. At least I know my config is sound, and I just did not have any test. Otherwise the user my doubt Test-Kitchen or Kitchen-Pester...